Over21/controllers/ViewController.swift

```diff
@@ -382,3 +412,99 @@ extension ViewController: CaptureHelperDeviceDecodedDataDelegate {
+        NotificationCenter.default.addObserver(self, selector: #selector(didBTDeviceList(notification:)), name: NSNotification.Name(rawValue: __NOTIFICATION_NAME_BT_FOUND_PRINTER_), object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(didBTComplete(notification:)), name: NSNotification.Name(rawValue: __NOTIFICATION_NAME_BT_LOOKUP_COMPLETE_), object: nil)
+    }
+    
+    func btLookup() {
+        printerCon?.refreshBTLookup()
+    }
+    
+    @objc func didBTStart(notification: NSNotification) {
+        
+    }
+    
+    @objc func didBTComplete(notification: NSNotification) {
+        
+    }
+    
+    @objc func didBTDeviceList(notification: NSNotification) {
+        if let _ = notification.userInfo {
+            if let lookupDevice = notification.userInfo?[__NOTIFICATION_NAME_BT_FOUND_PRINTER_] as! UPOSPrinter? {
+                printerList?.addDevice(lookupDevice)
+                printerList?.save()
+                connect()
+            }
+        }
+    }
+    
+    func statusUpdateEvent(_ status: NSNumber!) {
+        
+    }
+    
+    func connect() {
+        let target = printerList?.getList()?.last as? UPOSPrinter
+        if target == nil {
+            return
+        }
+        
+        printerDevice = target
+        
+        var result = printerCon?.open(target?.modelName)
+        if result == nil || result! != 0 {
+            return
+        }
+        
+        result = printerCon?.claim(5000)
+        if result == nil || result != 0 {
+            return
+        }
+        
+        Thread.sleep(forTimeInterval: 0.1)
+        printerCon?.deviceEnabled = true
+        ageIndicatorView.updatePrinterConnection(isConnected: true)
+    }
+    
+    func disconnect() {
+        printerCon?.deviceEnabled = false
+        let result = printerCon?.releaseDevice()
+        
+        if result == nil || result != 0 {
+            return
+        }
+        
+        Thread.sleep(forTimeInterval: 0.1)
+        printerCon?.close()
+    }
+    
+    func printData() {
+        let givenName = map["DAC"] ?? ""
+        printToPrinter(data: "Age verified\n\n")
+        printToPrinter(data: "Hi \(givenName),\n" )
+        printToPrinter(data: "Please enjoy your 10%\noff coupon for your\ndrink.\n\n")
+        printBarCodeToPrinter(data: "1234567890123")
+        printToPrinter(data: "\n\n\n\n\n")
+        printToPrinter(data: "\u{1B}")
+    }
+    
+    func printToPrinter(data: String) {
+        print(data)
+        printerCon?.printNormal(Int(__UPOS_PRINTER_STATION.PTR_S_RECEIPT.rawValue), data: data)
+    }
+    
+    func printBarCodeToPrinter(data: String) {
+        print(data)
+        printerCon?.printBarcode(Int(__UPOS_PRINTER_STATION.PTR_S_RECEIPT.rawValue), data: data, symbology: 104, height: 60, width: 200, alignment: -2, textPostion: -13)
```

Comment:
we should create a constant for the symbology, because `104` does not say what it is...

Reply:
added constant